apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryDbType.cs

```diff
@@ -100,5 +101,38 @@ internal static class BigQueryDbTypeExtensions
         };
 
         internal static string ToParameterApiType(this BigQueryDbType type) => s_typeToNameMapping[type];
+
+        internal static Type ToClrType(this BigQueryDbType type)
```

Comment:
Maybe use a `Dictionary<BigQueryDbType, type>`?

Reply:
Considered that, but ultimately, it felt redundant (the declaration would be on order size top the switch) and not really adding anything perf-wise.

Reply:
Sure. No big deal, and it wouldn't be an API change if we decided to do it.